point_pattern.py

```diff
@@ -56,9 +57,15 @@ def find_largest_city(gj):
     population : int
                  The population of the largest city
     """
-    city = None
+    temp = gj['features']
+    city = ""
     max_population = 0
 
+    for i in temp:
+        if (i['properties']['pop_max'] > max_population):
+            max_population = i['properties']['pop_max']
+            city = i['properties']['name']
+
     return city, max_population
```

Comment:
I pulled maybe 30 days of geoloacated data off the twitter stream.  The approach I landed on was to use MongoDB (it has a good Python API) to store the data (the raw tweet).  Then I analyzed the data in an iPython notebook using a combination of MongoDB database queries and raw Python.  That might be a viable workflow that preserves the raw data?